routers/api/v1/api.go

```diff
@@ -956,14 +956,11 @@ func Routes() *web.Route {
 				Get(user.ListEmails).
 				Post(bind(api.CreateEmailOption{}), user.AddEmail).
 				Delete(bind(api.DeleteEmailOption{}), user.DeleteEmail)
-
-			// create or update a user's actions secrets
 			m.Group("/actions/secrets", func() {
 				m.Combo("/{secretname}").
 					Put(bind(api.CreateOrUpdateSecretOption{}), user.CreateOrUpdateSecret).
-					Delete(repo.DeleteSecret)
+					Delete(user.DeleteSecret)
```

Comment:
Is this a bug fix? If that, maybe it should be backport.

Reply:
Yes indeed it is a fixed bug. Let me move it to another PR

Reply:
Made a separate PR for it here : https://github.com/go-gitea/gitea/pull/27829